include/clink/feature/one_hot_encoder.h

```diff
@@ -39,21 +44,27 @@ class OneHotEncoderModel {
   OneHotEncoderModel &operator=(const OneHotEncoderModel &) = delete;
 
   // Converts the provided data to a SparseVector.
-  llvm::Expected<SparseVector> transform(const int value,
-                                         const int columnIndex);
+  tfrt::AsyncValueRef<SparseVector> transform(const int value,
```

Comment:
Could we add a TODO to add a transform method with generic signature in the `Model` class?

Reply:
OK. I'll add a TODO here.

Same question also exists for `Vector` and `SparseVector`, but this one is easier to implement. I'll add the get/set methods to `Vector` in this PR.